src/tomlToJSON.cc

```diff
@@ -0,0 +1,130 @@
+/* ========================================================================== *
+ *
+ * @file tomlToJSON.cc
+ *
+ * @brief Convert a TOML string to a JSON object.
+ *
+ *
+ * -------------------------------------------------------------------------- */
+
+#include <string>
+
+#include <nlohmann/json.hpp>
+#include <toml.hpp>
+
+#include "flox/core/exceptions.hh"
+
+
+/* -------------------------------------------------------------------------- */
+
+namespace flox {
+
+/* -------------------------------------------------------------------------- */
+
+  nlohmann::json
+tomlToJSON( std::string_view toml )
+{
+  std::string        tomlStr( toml );
+  std::istringstream tomlStream( tomlStr );
+
+  std::function<void( nlohmann::json &, const toml::value & )> visit;
+
+  visit = [&]( nlohmann::json & jto, const toml::value & tfrom )
```

Comment:
This aligns with the `nix` parsing behavior and uses the same library.

The only differences is that instead of parsing to `nix::Value` types, I emit `nlohmann::json` types.